### PR TITLE
ci: use built in black profile for isort

### PR DESCRIPTION
**Pull request by jon-hagg**
### Purpose
We can take advantage of isort's built in "profile" for black compatibility. This includes a line length option which we were msising, along with a couple others I'm not sure about. [Source](https://github.com/PyCQA/isort/blob/develop/isort/profiles.py) from their repo is easy to read.

### What the code is doing
Added isort section in tox.ini

### Testing
Ran `tox -e format` locally, and ci passes.

### Time estimate
5 min

### CODE REVIEW

**Review by BainanXia — APPROVED** [review]
Nice, I believe this will resolve the conflicts between `isort` and `black` that @danielolsen encountered before.

File: tox.ini [hunk 1]
```diff
@@ -15,10 +15,13 @@ commands =
     pytest: pipenv sync --dev
     pytest: pytest
     format: black .
-    format: isort -m 3 --tc .
```
Comment (danielolsen) [inline tox.ini:18]: What are these options that we're dropping?

Comment (jon-hagg) [reply tox.ini:18]: The `-m 3` is `multi_line_output = 3` and `--tc` is `include_trailing_comma = True`

Comment (jon-hagg) [reply tox.ini:18]: They are both included in the profile, which we were trying to reproduce earlier but was incomplete

Comment (danielolsen) [reply tox.ini:18]: Ah, and black probably already incorporates these, yeah?

**Review by danielolsen — APPROVED** [review]
Thanks!

**Pull request merged**